app/assets/javascripts/discourse/app/lib/plugin-api.gjs

```diff
@@ -333,6 +334,18 @@ class PluginApi {
     return klass;
   }
 
+  addBehaviorTransformerName(name) {
+    _addTransformerName(name, transformerTypes.BEHAVIOR);
+  }
+
+  registerBehaviorTransformer(transformerName, behaviorCallback) {
```

Comment:
Would be good to add some docs/examples in a comment above this function

Reply:
Done